paper/paper.tex

```diff
@@ -0,0 +1,268 @@
+
+For example, consider the NHANES dataset, which includes clustering and stratification. The following example demonstrates how to create a \verb|SurveyDesign| object for this dataset:
+\begin{lstlisting}
+julia> nhanes = load_data("nhanes"); 
+# CSV dataframe included with the package
+
+julia> design = SurveyDesign(nhanes;
+                    clusters=:SDMVPSU,
+                    strata=:SDMVSTRA, 
+                    weights=:WTMEC2YR);
+    \end{lstlisting}
+Consider another example, a cluster sample based on the Academic Performance Index for all California schools based on standardised testing of students. There is no stratification in this example. 
+
+\begin{lstlisting}
+julia> apiclus1 = load_data("apiclus1"); 
+# CSV dataframe included with the package
+
+design = SurveyDesign(apiclus1; 
+            clusters=:dnum, weights=:pw);
+    \end{lstlisting}
+% \begin{lstlisting}
+% julia> nhanes = load_data("nhanes") 
+% # CSV dataframe included with the package
+
+% julia> SurveyDesign(nhanes; clusters=:SDMVPSU,
+%                     strata=:SDMVSTRA, 
+%                     weights=:WTMEC2YR)
+
+% SurveyDesign:
+% data: 8591 x 11 DataFrame
+% strata: SDMVSTRA
+%     [83, 84, 86  ...  81]
+% cluster: SDMVPSU
+%     [1, 1, 2  ...  2]
+% popsize: [244586.316, 43527.8366, 36124.9061  ...  19331.022]
+% sampsize: [3, 3, 3  ...  3]
+% weights: [81528.772, 14509.2789, 12041.6354  ...  6443.674]
+% allprobs: [0.0, 0.0001, 0.0001  ...  0.0002]
+% \end{lstlisting}
+\section{Estimation}
+
+Survey.jl provides a range of estimators for survey data analysis. These include univariate statistics such as mean, median, total, and quantiles, as well as multivariate statistics such as regressions and ratios. For example, to estimate the mean of the \verb|:api99| column in the \verb|design| SurveyDesign:
+
+\begin{lstlisting}
+julia> mean(:api99, design)
+1x1 DataFrame
+    Row | mean    
+        | Float64 
+--------|--------
+      1 | 606.978
+    \end{lstlisting}
+This command estimates the mean of column \verb|:api99|.
+
+For multivariate statistics such as regressions\footnote{Regressions are performed using GLM.jl. Instead of passing a DataFrame, a survey design is passed to the function, maintaining a familiar interface. This approach of using multiple dispatch is applied to all estimators imported from other packages, ensuring consistency and ease of use.}:
+
+\begin{lstlisting}
+julia> glm(@formula(y ~ x),
+            my_design, Normal(), IdentityLink())
```

Comment:
A semicolon missing? Since we're not showing any output?

Reply:
> A semicolon missing? Since we're not showing any output?

```suggestion
            my_design, Normal(), IdentityLink()); 
```